content/articles/where-are-your-servers-located.markdown

```diff
@@ -14,4 +14,4 @@ Our Anycast network consists of 6 points-of-presence around the world:
 * Washington DC, US
 * Amsterdam, Netherlands
 * Tokyo, Japan
-- Sydney, Australia
```

Comment:
I'm with you, I prefer `-`. 😄

Reply:
Maybe I'll change all our lists to `-`s in a different PR ;p